openhands/mcp/utils.py

```diff
@@ -52,6 +62,7 @@ async def create_mcp_clients(
     sse_servers: list[MCPSSEServerConfig],
     shttp_servers: list[MCPSHTTPServerConfig],
     conversation_id: str | None = None,
+    stdio_servers: list[MCPStdioServerConfig] | None = None,
```

Comment:
Can we move this to the third argument? Putting this after `conversation_id` feels a bit werid